disk/src/com/treode/disk/messages/package.scala

```diff
@@ -38,3 +39,9 @@ package messages {
   case class AlreadyDraining (drive: Path) extends Message {
     def en = s"Already draining: ${quote (drive)}"
   }}
+
+package object messages {
+
+  implicit def stringToPath (path: String): Path =
```

Comment:
One must be very careful with implicit defs. You'll notice that I've kept them largely confined to testing, and to get them you must import a narrow set of test tools. In other words, don't do this here. Instead, you can define it in SpecTools that's a sibling of DrivesAttachHandlerSpec.